### PR TITLE
fix#11221: added missing version page for services

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes:

Partial changes for #11221 

I worked on following

- [x] Added version page for all the services
- [x] Replaced existing custom version timeline drawer with `antd` Drawer component
- [x] Added unit and cypress tests for newly added pages

https://github.com/open-metadata/OpenMetadata/assets/51777795/09bc1d77-400f-430a-8146-68a9b91aebb8

https://github.com/open-metadata/OpenMetadata/assets/51777795/e40de5aa-0412-49ba-ae98-28a827c39d83

#
### Type of change:

- [x] Improvement
- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] My PR title is `Fixes <issue-number>: <short explanation>`
- [x] I have commented on my code, particularly in hard-to-understand areas. 
- [x] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/constants/Version.constants.js [hunk 1]
```diff
@@ -495,96 +505,98 @@ const CONTAINER_PATCH_PAYLOAD = [
...
+    value: `Description for ${CONTAINER_NAME}`,
   },
 ];
 
 export const ENTITY_DETAILS_FOR_VERSION_TEST = {
   Table: {
-    name: 'cypress_version_test_table',
-    term: 'cypress_version_test_table',
+    name: TABLE_NAME,
+    term: TABLE_NAME,
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/constants/Version.constants.js:515]: What's term?

File: openmetadata-ui/src/main/resources/ui/src/constants/constants.ts [hunk 2]
```diff
@@ -384,6 +385,21 @@ export const getVersionPathWithTab = (
...
+export const getServiceVersionPath = (
+  serviceCategory: string,
+  serviceFQN: string,
+  version: string
+) => {
+  let path = ROUTES.SERVICE_VERSION;
+
+  path = path
+    .replace(PLACEHOLDER_ROUTE_SERVICE_CAT, serviceCategory)
+    .replace(PLACEHOLDER_ROUTE_SERVICE_FQN, serviceFQN)
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/constants/constants.ts:397]: Why constant has util function

File: openmetadata-ui/src/main/resources/ui/src/pages/ServiceVersionPage/ServiceVersionMainTabContent.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  serviceName: string;
+  serviceDetails: ServicesType;
+  data: ServicePageData[];
+  isServiceLoading: boolean;
+  paging: Paging;
+  currentPage: number;
+  pagingHandler: (cursorType: string | number, activePage?: number) => void;
+  entityType: EntityType;
+  changeDescription: ChangeDescription;
+}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ServiceVersionPage/ServiceVersionMainTabContent.tsx:51]: .interface file

File: openmetadata-ui/src/main/resources/ui/src/pages/ServiceVersionPage/ServiceVersionPage.tsx [hunk 4]
```diff
@@ -0,0 +1,510 @@
...
+      data: ServicesType,
+      titleBreadCrumb: TitleBreadcrumbProps['titleLinks']
+    ) => {
+      setTier(getTierTags(tags));
+      setOwner(owner);
+      setCurrentVersionData(data);
+      setBreadcrumbLinks(titleBreadCrumb);
+    },
+    []
+  );
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ServiceVersionPage/ServiceVersionPage.tsx:151]: Can we avoid soo many setState?

**Comment by aniketkatkar97** [comment]
@chirag-madlani pushed the changes as per the comments. Can you please review it again?

File: openmetadata-ui/src/main/resources/ui/src/pages/ServiceVersionPage/ServiceVersionPage.tsx [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+
+  const entityType = useMemo(
+    () => getEntityTypeFromServiceCategory(serviceCategory),
+    [serviceCategory]
+  );
+
+  const resourceEntity = useMemo(
+    () => getResourceEntityFromServiceCategory(serviceCategory),
+    [serviceCategory]
+  );
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ServiceVersionPage/ServiceVersionPage.tsx:125]: we can use single useMemo here

**Review by chirag-madlani — DISMISSED** [review]

**Comment by aniketkatkar97** [comment]
@chirag-madlani pushed the suggested changes

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**